encrypted-config-value-module/build.gradle

```diff
@@ -6,7 +6,6 @@ dependencies {
     api 'com.google.guava:guava'
     api 'com.fasterxml.jackson.core:jackson-databind'
     api 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
-    api 'org.apache.commons:commons-text'
```

Comment:
I think it's worth the possibility of someone having used this. Hopefully our tooling would have made them put a direct dep on it anyway.

Reply:
Yep, I considered making this PR but keeping the commons-text dep as `runtimeOnly` to avoid abi breaks, but I don't think it's worth the trouble because it retains the entirety of the downside without changing much from our current behavior.